point-of-care-poc/work-flows/traige.md

```diff
@@ -1,12 +1,23 @@
 Patient check-in is the process that sends a patient from one service provider to another:
 To Check-in a patient
-   1. Log into the system using the Reception Location
-   2. Click the Find Patient Icon as shown below 
-   ![Register Patient Link](/assets/poc_find_patient_link.png)
-   3. In the Patient Search bar, enter the patient names or any other patient identifier and wait for the results to display.
-   4. On the Patient record displayed, click the Check-in icon under Actions
-   ![](/assets/patient_record_poc.png)
-   5. A check-in Dialogue box will be displayed, as shown below.![](/assets/checkin_poc.png)
-   6. Enter all the Required information for in this dialogue box and click **Check-in** to Check-in a patient or **Cancel** to cancel the check-in.
-   7. Repeat the process for all patients that must be checked in on that particular day.
+
+1.Log into the system using the Reception Location
+
+2.Click the Find Patient Icon as shown below 
+   
+![Register Patient Link](/assets/poc_find_patient_link.png)
```

Comment:
@ODORA0 The images do not appear

Reply:
@ssmusoke How come, becasue this renders well on my markdown preview on the IDE

Reply:
@ssmusoke Could you try again to see if the images render for you